test/es-module/test-esm-loader-programmatically.mjs

```diff
@@ -1,14 +1,35 @@
-import '../common/index.mjs';
 import * as fixtures from '../common/fixtures.mjs';
-
+import assert from 'node:assert';
+import { execPath } from 'node:process';
 import { describe, it } from 'node:test';
-import { registerLoader } from 'node:module';
+import { spawnPromisified } from '../common/index.mjs';
 
 // This test ensures that the registerLoader function can register loaders
 // programmatically.
 
-describe('Loader programmatically', { concurrency: true }, () => {
-  it('are called with all expected arguments', async () => {
-    await registerLoader(fixtures.path('/es-module-loaders/loader-resolve-foo.mjs'));
```

Comment:
Please don’t delete any existing tests. You should create a new test file for this feature, like `test/es-module/test-esm-register-loader.mjs`

Reply:
Sorry Geoffrey, actually this [test-esm-loader-programmatically.mjs](https://github.com/nodejs/node/pull/46826/files/4dd44418c24509efef9ea0948ff5fe8a68387d6c..daf99317bb78ca8026957d98ef428b68fbd9b3cd#diff-16536390c527c3f1d1d485d8fbc8b621a3269adbfbcb795104127194a1d56d3e) file was created by me in old commits of this PR. I think I might have deleted it for some reason instead of rewriting the content.